UKHomeOffice/sgmr-service
# NMSW-34 Remove old reach autocomplete

## AC

Remove use of @reach/combobox autocomplete component

----

## To test

- delete your `node_modules` folder
- `npm ci`
- `npm tests` all should pass
- `npm run lint` all should pass

----

## Notes
